Lab1.ipynb

```diff
@@ -241,15 +249,256 @@
+      "text/plain": [
+       "<Figure size 1080x720 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "n = 10\n",
+    "p = .9\n",
+    "# defining list of r values \n",
+    "r_values = list(range(n + 1)) \n",
+    "# list of pmf values \n",
+    "dist = [binom.pmf(r, n, p) for r in r_values ] \n",
+    "# plotting the graph  \n",
+    "plt.bar(r_values, dist) \n",
+    "plt.xlabel('X', fontsize = 27) \n",
+    "plt.ylabel('P(x)', fontsize = 27)\n",
+    "plt.title('Bionomial Distribution w/ n = 10, p = .9', fontsize = 30)\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Seen in the code above with a binomial distrbution, it's almost impossible to succeed less than five times in a 10 trial grouping. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "C: Does a greater sample size of 15 have a larger expetected value than a sample size of 10?"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 168,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "5.0\n",
+      "8.0\n"
+     ]
+    }
+   ],
+   "source": [
+    "n = 10\n",
+    "p = .5\n",
+    "print (stats.binom(n, p).ppf(.5))\n",
+    "n = 15\n",
+    "print (stats.binom(n, p).ppf(.5))"
```

Comment:
Part 2 Problem 1C:
- this might be better as support for your explanations of binomial distribution characteristics in Problem 1B above, below your charts